dir.props

```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```

Comment:
><When Condition="'$(TargetGroup)'=='netcoreapp1.1'"> [](start = 4, length = 52)

Given netstandard1.7 resolves to netcoreapp1.1, should we also add a `<When>` block for netstandard1.7 here?